packages/components/Checkbox/src/Checkbox.types.ts

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { IPressableProps } from '@fluentui-react-native/pressable';
+import { IPressableState } from '@fluentui-react-native/interactive-hooks';
+import { ViewProps } from 'react-native';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { ITextProps } from '@fluentui-react-native/text';
+import { ITextTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens } from '@fluentui-react-native/tokens';
+import { IViewWin32Props } from '@office-iss/react-native-win32';
+
+export const checkboxName = 'Checkbox';
+
+export interface ICheckboxInfo extends IPressableState {
+  checked: boolean;
+
+  disabled: boolean;
+
+  // To allow overrides in .settings. 'start' || undefined = false and 'end' = true
+  boxSide: boolean;
+}
+
+export interface ICheckboxState {
+  info: ICheckboxInfo;
+}
+
```

Comment:
Ok I see, I was a bit confused on that. So in this case, since the only thing that's needed in RenderData from state is boxSide, but boxSide has an impact on settings overrides, info is completely unnecessary here?

Reply:
Changed in newest iteration. Resolving thread